### PR TITLE
✨ feat: Created home landing page #1

### PR DESCRIPTION
**Pull request by Makig10**
# Description
Created the home landing page to resolve issue 1
Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Reference the issue related to this PR as shown below. Every issue has it's own number you can find the issue numbers [here](https://github.com/italanta/italanta-apps/issues).

Fixes # (issue) - *Example*: Fixes #10, #20 NOTE: GitHub issue numbers should be prefixed with **#** to aid with issue tracking.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**: (optional)

- Firmware version:
- Hardware:
- Toolchain:
- SDK:

## Checklist

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ x] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,12 +8,61 @@
...
+    </div>
+
+    <div id="footer">
+      <img>
+      <img>
+      <img>
+      <img>
+      <img>
+      <img>
+      <img>
```
Comment (JAPHETHNYARANGA) [inline index.html]: ?? whats going on here

**Review by JAPHETHNYARANGA — CHANGES_REQUESTED** [review]
Use lipstick emoji for ui elements. Please make the page responsive also and share screen shots.

**Comment by Makig10** [comment]
Ok, will do that.

**Comment by JAPHETHNYARANGA** [comment]
closing as this is done, please pick another issue

**Comment by Makig10** [comment]
Awesome, thanks for the update!

**Pull request closed without merging**